zio-sbt-ecosystem/src/main/scala/zio/sbt/ScalaCompilerSettings.scala

```diff
@@ -338,4 +338,14 @@ trait ScalaCompilerSettings {
   def addOptionsOnExcept(scalaBinaryVersions: String*)(options: String*): Def.Setting[Task[Seq[String]]] =
     addOptionsOn(Seq("2.12", "2.13", "3").diff(scalaBinaryVersions)*)(options*)
 
+  private def betterMonadicForSettings =
+    Seq(
+      libraryDependencies ++= {
+        CrossVersion.partialVersion(scalaVersion.value) match {
+          case Some((2, _)) => Seq(compilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"))
```

Comment:
I think it's better to reference this module inside a source file which can be detected by tools like scala-steward and renovate.

Reply:
I made the change